src/types/redis_bitmap.cc

```diff
@@ -335,14 +341,39 @@ rocksdb::Status Bitmap::BitPos(const Slice &user_key, bool bit, int64_t start, i
     return -1;
   };
 
+  auto bit_pos_in_byte_startstop = [](char byte, bool bit, uint32_t start, uint32_t stop) -> int {
+    for (uint32_t i = start; i <= stop; i++) {
+      if (bit && (byte & (1 << i)) != 0) return (int)i;  // typecast to int since the value ranges from 0 to 7
```

Comment:
`// typecast to int since the value ranges from 0 to 7`

What does this mean? Isn't something like uint8_t or other is proper? uint32_t and int ( usally int32_t ) doesn't have significant different here?